src/components/Proposal/Create/SchemeForms/CreateContributionRewardProposal.tsx

```diff
@@ -229,12 +234,14 @@ class CreateContributionReward extends React.Component<IProps, IStateProps> {
                 className={touched.title && errors.title ? css.error : null}
               />
 
-              <label htmlFor="descriptionInput">
+              <TrainingTooltip overlay={fnDescription} placement="right">
```

Comment:
why is this a function?

Reply:
for lint and because it needs to return a component.